analyzer-java/src/main/java/com/infosupport/ldoc/analyzerj/descriptions/TypeDescription.java

```diff
@@ -52,7 +52,9 @@ public record TypeDescription(
 
     implements Description {
 
-  /** Builder class for constructing TypeDescriptions incrementally. */
+  /**
+   * Builder class for constructing TypeDescriptions incrementally.
+   */
```

Comment:
I would say these changes are also unneeded, so would suggest reverting them.

The reason I keep asking about these is that I do these reviews line-by-line, so if it _looks_ like something has changed but actually it hasn't, that means reviews take longer than necessary.

Reply:
problem is i have to manually revert these as my code formatter likes adding new lines to the javadoc.

Reply:
If you use IDEA, you can make sure the following checkbox is **checked**:

File > Settings > Editor > Code Style > Java > JavaDoc > Other > "Do not wrap one line comments"

But in general, I do recommend going line-by-line over your own pull requests as well and making sure the changes you're submitting are the ones you intend to submit.